### PR TITLE
CPLAT-10392: v1.5.0

### PR DESCRIPTION
**Pull request by evanweible-wf**
Changes from #58 with an additional commit to update the changelog.

### CODE REVIEW

**Comment by aviary-wf** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://dev.workiva.net/docs/teams/information-security/aviary#configuring-aviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: lib/src/constants.dart [hunk 1]
```diff
@@ -5,6 +5,10 @@ final RegExp importExportDartPackageRegex =
 /// Regex used to detect all Sass import directives.
 final RegExp importScssPackageRegex = RegExp(r'''\@import\s+['"]{1,3}package:\s*([a-zA-Z0-9_]+)\/[^;]+''');
 
+/// Regex used to detect all Less import directives.
+final RegExp importLessPackageRegex =
+    RegExp(r'@import[\t ]+(?:\(.*\)\s+)?"(?:packages\/|package:\/\/)([a-zA-Z1-9_-]+)\/');
```
Comment (seanburke-wf) [inline lib/src/constants.dart:10]: Is there a reason to use `[\t ]+` after `@import` instead of `\s+`?

File: test/utils_test.dart [hunk 2]
```diff
@@ -104,6 +104,52 @@ include: package:pedantic/analysis_options.1.8.0.yaml
...
+        sharedTest('@import "package://_foo/bar";', '_foo');
+      });
+    });
+
+    test('with extra whitespace in the line', () {
+      sharedTest('   @import   "packages/foo/bar"   ;   ', 'foo');
+      sharedTest('   @import   "package://foo/bar"   ;   ', 'foo');
+    });
+
+    test('with multiple import\'s in the same line', () {
```
Comment (seanburke-wf) [inline test/utils_test.dart:140]: #nit: "import's" -> "imports"

File: lib/src/constants.dart (hunk 1 above)
Comment (evanweible-wf) [reply lib/src/constants.dart:10]: I'm not familiar with LESS so I'll defer to @Anrock

Comment (Anrock) [reply lib/src/constants.dart:10]: I'm not really familiar with less either, but from the spec it seemed like no newlines were allowed after `@import` and `s` includes newlines. So I've used `[\t ]` to allow only non-linebreaking whitespace.

Comment (seanburke-wf) [reply lib/src/constants.dart:10]: My biggest concern here was other whitespace (non-breaking space or zero-width, for example), but playing with http://lesscss.org/less-preview/ it does look like newline is allowed.

**Comment by evanweible-wf** [comment]
@seanburke-wf addressed your comments.

**Comment by evanweible-wf** [comment]
QA +1
@Workiva/release-management-p

**Pull request merged**